src/store/actionCreators.js

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
```

Comment:
This is a topic for debate, however technically, "actionCreator" is a function that returns an "action".
In this file each function is a "thunk" (thunk - a function, that adds delay, wrapped around another function with the actual logic).